src/components/PersonalSettings.vue

```diff
@@ -106,6 +109,12 @@
 							<td v-else>
 								{{ quota.used + ' ' + quota.unit }}
 							</td>
+							<td v-if="quota.used_pool">
+								{{ quota.limit > 0 ? Math.round(quota.used_pool / quota.limit * 100) + ' %' : quota.used_pool + ' ' + quota.unit }}
```

Comment:
quota.unit should be translated like more or less:
``` js
quota.unit === 'day' 
    ? n('integration_openai', '%s day', '%s days', number)
    : n('integration_openai', '%s month', '%s months', number)
```

This should probably be in a computed prop.